### DISCUSSION TITLE
View total time printed after print has finished and backlight was turned off?

### DISCUSSION THREAD

**Discussion by GT-Geo** [mriscoc/Ender3V2S1#214] [Q&A] [answered]
I turn off the screen during long prints, when the print is complete I hit the button to turn the screen back on but don't see how long the print took. I have to click print again to get the estimate which is not always accurate.

Is there a way to display how long the print took from the main screen?

**Comment by mriscoc** [comment]
You can dim the screen backlight, the DWIN display never turns off, the firmware controls the backlight, when you select turn off backlight it is set to 0, but you can lower the backlight to 5.

**Reply by GT-Geo** [reply]
However, I notice that after the long print and when I turn the screen back on (or the backlight is turned on) the printed time flashes on the screen briefly but disappears. 

You can pause this video at the beginning to see the screen in question flash quickly then disappear.
https://www.youtube.com/shorts/s6cLif_tyZU

Can you correct this so the information remains on the screen with the next update?

**Comment by mriscoc** [comment, marked as answer]
In the latest version you are able to turn on the backlight by rotating the knob, then the print done screen will be there.
